src/Symfony/Component/Finder/Finder.php

```diff
@@ -567,17 +567,17 @@ public function followLinks()
      *
      * @return $this
      */
-    public function ignoreUnreadableDirs($ignore = true)
+    public function ignoreUnreadableDirs(bool $ignore = true)
     {
-        $this->ignoreUnreadableDirs = (bool) $ignore;
+        $this->ignoreUnreadableDirs = $ignore;
 
         return $this;
     }
 
     /**
      * Searches files and directories which match defined rules.
      *
-     * @param string|array $dirs A directory path or an array of directories
+     * @param string|string[] $dirs A directory path or an array of directories
```

Comment:
docblock fixes should also be applied to lower branches (i.e. `3.4` here)

Reply:
Okay, I will provide a PR with both docblock fixes